.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,82 @@
-<!-- Please feel free to remove whatever sections/lines in this aren’t relevant.
+<!---
+Welcome! Thank you for contributing to the U.S. Web Design System. Your contributions are vital to our success and we are glad you're here.
 
-Use the title line as the title of your pull request, then delete these lines.
+Please keep in mind:
+- This pull request (PR) template exists to help speed up integration. The USWDS Core team reviews and approves every PR before merging it into the public code base, so the better we understand the problem and solution, the sooner we can merge this change. The point here is: clear explanations matter!
+- You can see examples of successful PRs that meet our criteria by ... [TK?]
+- It can be helpful to annotate code changes with GitHub comments.
+- You can erase any part of this template that doesn't apply to your pull request (including these instructions!).
 
-## Title line template: [Title]: Brief description
+If you get stuck, you can find more information about contributing in [contributing.md](https://github.com/uswds/uswds/blob/develop/CONTRIBUTING.md) or you can reach out to us directly at uswds@gsa.gov.
+ -->
 
-Website: For pull requests that impact designsystem.digital.gov’s look, feel, or functionality, please open a pull request on the uswds-site repo (https://github.com/uswds/uswds-site).
+<!---
+Step 1
+Title this PR with the following format:
+USWDS - [Package]: [Brief statement describing what this pull request solves]
+eg: "USWDS - Button: Increase font size"
+ -->
 
+## Summary
+<!--
+Provide a one or two sentence summary of the update that can be used in the changelog.
```

Comment:
I like this suggestion. I have added summary text outside the comments in each section so that users can see a simpler (but still helpful) view on preview.
Added in [3b26fab](https://github.com/uswds/uswds/pull/4842/commits/3b26fab71d8072acee70845defff4754899bdd15)